### DISCUSSION TITLE
Feature Suggestion: Grid wiki/comments

### DISCUSSION THREAD

**Discussion by kerridge0** [djangopackages/djangopackages#671] [Grids]
How about a further information section for each grid, 

it could be a comments section with up/down voting, 

or a "wikipedia style" page of editorial

People could add links to comparative summaries / walk-throughs / reviews relating to one or more packages in the grid?

This could be a useful way for people to contribute to the site as well as a really good way to help people choose between packages in a grid.

also, possibly back-links?

**Comment by ryancheley** [comment]
It looks like the original reporter is no longer active on GitHub and there haven't been any new comments in more than 10 year. Marking as closed
